cmd/limactl/copy.go

```diff
@@ -117,9 +128,17 @@ func copyAction(cmd *cobra.Command, args []string) error {
 
 	sshCmd := exec.Command(arg0, append(sshArgs, scpArgs...)...)
 	sshCmd.Stdin = cmd.InOrStdin()
-	sshCmd.Stdout = cmd.OutOrStdout()
-	sshCmd.Stderr = cmd.ErrOrStderr()
-	logrus.Debugf("executing scp (may take a long time)): %+v", sshCmd.Args)
+	if quiet {
+		sshCmd.Stdout = nil
+		sshCmd.Stderr = nil
+	} else {
+		sshCmd.Stdout = cmd.OutOrStdout()
+		sshCmd.Stderr = cmd.ErrOrStderr()
+	}
```

Comment:
Can be simplified:
```suggestion
	if !quiet {
		sshCmd.Stdout = cmd.OutOrStdout()
		sshCmd.Stderr = cmd.ErrOrStderr()
	}
```

The fields are `nil` by default.